src/View/Helper/FormHelper.php

```diff
@@ -359,12 +366,16 @@ public function create($model = null, array $options = [])
             'encoding' => strtolower(Configure::read('App.encoding')),
             'templates' => null,
             'idPrefix' => null,
+            'valuesSources' => $this->getValuesSources(),
```

Comment:
This was 'context' which is the default.
It is now set to the singleton state, e.g .if changed once, further `Form::create()` calls will be based on the changed state.